.github/actions/setup_environment/action.yml

```diff
@@ -67,7 +67,7 @@ runs:
       shell: bash
       run: |
         python -m pip install --upgrade pip
-        pip install -e .[server,templates,secrets-aws,secrets-gcp,secrets-azure,secrets-hashicorp,dev]
+        pip install -e .[server,templates,secrets-aws,secrets-gcp,secrets-azure,secrets-hashicorp,s3fs,gcsfs,adlfs,dev]
```

Comment:
Technically it should work because GCP/Azure artifact stores only import and use those dependencies. The other integration dependencies should not get imported after the changes Baris and I made when we refactored all the integrations for the ZenML dashboard release.

However, I actually did not test this manually with GCP/Azure yet. I guess I should do that regardless before we merge.